shortnames.conf

```diff
@@ -8,6 +8,7 @@
   "archlinux" = "docker.io/library/archlinux"
   # centos
   "centos" = "quay.io/centos/centos"
+  "centos-bootc" = "quay.io/centos-bootc/centos-bootc"
```

Comment:
Let's not do this one right now as it is likely to change in the relatively near future by moving to `quay.io/centos` I hope.

Reply:
There is also no :latest version, which will cause tests to fail.

Reply:
Yeah...that's a legacy of the centos/centos-stream split I think.  There *is* a `quay.io/centos/centos:latest` but it's actually last built 
`"Created": "2020-12-08T00:22:53.076477777Z"`

Probably it's time to just replace `centos:latest` with `centos:stream9` for now...

Reply:
Who do we contact to get quay.io/centos/centos-bootc in place?